modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/30_search.yml

```
@@ -482,3 +482,100 @@
                         }]
     - match: { error.root_cause.0.type: "illegal_argument_exception" }
     - match: { error.root_cause.0.reason: "script score function must not produce negative scores, but got: [-9.0]"}
+
+---
+
+"Flat-object fields from within the scripting":
+  - skip:
+      version: " - 2.99.99"
+      reason: "flat_object is introduced in 3.0.0 in main branch"
```

A reminder to change the condition for running the rest api test.

```suggestion
      version: " - 2.6.99"
      reason: "flat_object is introduced in 2.7.0"
```